api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -588,7 +620,64 @@ public Locale getLocale() {
 		return Context.getLocale();
 	}
 	
+	/**
+	 * Change the current locale.
+	 * 
+	 * @param locale The locale.
+	 */
 	public void setLocale(Locale locale) {
 		this.locale = locale;
 	}
+	
+	/**
+	 * @return the value of the Global Propriety Handle Timezones <
+	 */
+	public boolean convertTimezones() {
+		return BooleanUtils.toBoolean(
+		    Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_TIMEZONE_CONVERSIONS));
+		
+	}
+	
+	public String getJSDatetimeFormat() {
+		return Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_JS_DATETIME_FORMAT);
+	}
+	
+	public String getJSDateFormat() {
+		return Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_JS_DATE_FORMAT);
+	}
+	
+	public String getDatetimeFormat() {
+		return Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_DATETIME_FORMAT);
+	}
+	
+	public String getDateFormat() {
+		return Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_DATE_FORMAT);
+	}
+	
+	public String getTimeFormat() {
+		return Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_TIME_FORMAT);
+	}
+	
+	/**
+	 * @return the value of the User Propriety clientTimezone, that indicates the client timezone
+	 */
+	public String getClientTimezoneProperty() {
+		return Context.getAuthenticatedUser().getUserProperty("clientTimezone");
+	}
+	
+	/**
+	 * Change the user propriety clientTimezone, that have the value of the user timezone.
+	 * 
+	 * @param clientTimezone The client timezone.
+	 */
+	public void setClientTimezoneProperty(String clientTimezone) {
```

Comment:
I would just call this `setClientTimezone`. The fact that it is stored in a user property is an implementation detail.